### PR TITLE
1.19.2 dungeon chest patch

### PR DESCRIPTION
**Pull request by Christofmeg**
Added Ancient City and Ancient City Ice Box to Dungeon Chest Category

Updated lang files (there may be some wrong translations, I used a translator for most of them, hopefully they are correct)

Updated version

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -41,4 +41,4 @@ curseProjectId=240630
 curseHomepageLink=https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer
 
 # Version
-specificationVersion=1.2.2
\ No newline at end of file
+specificationVersion=1.2.2.201
```
Comment (way2muchnoise) [inline gradle.properties:44]: Please remove .201, this part will be added automatically by Travis when compiling and publishing the mod

File: Common/src/main/java/jeresources/compatibility/minecraft/MinecraftCompat.java [hunk 2]
```diff
@@ -102,20 +102,20 @@ private void registerVanillaPlants() {
         PlantDrop poisonous = new PlantDrop(new ItemStack(Items.POISONOUS_POTATO), 0.02F);
         registerPlant(new PlantEntry((PotatoBlock) Blocks.POTATOES, potato, poisonous));
 
-        //Carrot
+        // Carrot
```
Comment (way2muchnoise) [inline Common/src/main/java/jeresources/compatibility/minecraft/MinecraftCompat.java:105]: can you undo just adding the spaces? I don't really see the need for change in this file just for formatting of a comment

**Review by way2muchnoise — CHANGES_REQUESTED** [review]
Added few comments, thanks for the addition. Please review and also edit them in your other open PRs.

File: gradle.properties (hunk 1 above)
Comment (Christofmeg) [reply gradle.properties]: Okay thanks for clarifying that, will fix the same things in the other PRs aswell

File: Common/src/main/java/jeresources/compatibility/minecraft/MinecraftCompat.java (hunk 2 above)
Comment (Christofmeg) [reply Common/src/main/java/jeresources/compatibility/minecraft/MinecraftCompat.java]: :)

**Review by way2muchnoise — APPROVED** [review]

**Pull request merged**